src/sass/styles.scss

```
@@ -175,6 +178,20 @@ main {
   }
 }
 
+p.chapter {
+  font-size: 1.25em;
+  font-weight: 700;
+  margin-bottom: 2em; 
+  color: $lb-red;
+}
+
+p.title {
+  font-size: 2em;
+  font-weight: 700;
+  margin-bottom: 2em; 
+  color: $lb-black;
+}
+
```

```suggestion
p {
  .chapter {
    font-size: 1.25em;
    font-weight: 700;
    margin-bottom: 2em; 
    color: $lb-red;
  }

  .title {
    font-size: 2em;
    font-weight: 700;
    margin-bottom: 2em; 
    color: $lb-black;
  }
}

```

We can sassify this.